source/designers-guide/javascript-statemanager-and-pluginbase/index.md

```diff
@@ -370,6 +374,44 @@ StateManager.registerBreakpoint({
         * ```softError : Boolean``` - Truthy to return the provided property when no vendor was found, otherwise the method returns ```null```
     * Tests the provided CSS style property on an empty div with all vendor properties.
 
+## The EventEmitter
+The EventEmitter is a utility class, offering a simple base-class for event driven architecture. Currently the EventEmitter is used as the basis for the StateManager, but you can use it to let your own objects emit events.
+
+An object which inherits from the EventEmitter exposes the functionality of subscribing and listening to events. Much like the [global event observer](#global-jquery-event-observer).
+
+### Class methods
+* ```on()```
+    * **Arguments**
+        * ```eventName : String``` - The name of the event which to listen to
```

Comment:
Please use tabulator instead of whitespaces for the arguments.
If you use whitespaces, the listing of the arguments is incorrect (see preview mode).